javascript/index.js

```diff
@@ -15,51 +15,116 @@ let splits = document.getElementById('splits');
 
 function printTime() {
   // ... your code goes here
+  //callbacks of startClick prints the time, so you have to pass this when calling the startClick function
+
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
   // ... your code goes here
+  let min = chronometer.twoDigitsNumber(chronometer.getMinutes());
+  minDec.innerHTML = min[0];
+  minUni.innerHTML = min[1];
+  
 }
 
 function printSeconds() {
   // ... your code goes here
+  let seconds = chronometer.twoDigitsNumber(chronometer.getSeconds());
+  secDec.innerHTML = seconds[0];
+  secUni.innerHTML = seconds[1];
+ 
 }
 
 // ==> BONUS
 function printMilliseconds() {
   // ... your code goes here
+  let miliSeconds = chronometer.twoDigitsNumber(chronometer.getMiliSeconds());
+  milDec.innerHTML = miliSeconds[0];
+  milUni.innerHTML = miliSeconds[1];
 }
 
 function printSplit() {
   // ... your code goes here
+  let newLi = document.createElement('li');
+  newLi.innerText = chronometer.splitClick();
+  splits.appendChild(newLi);
+
 }
 
 function clearSplits() {
   // ... your code goes here
+  splits.innerHTML = '';
+
 }
 
 function setStopBtn() {
   // ... your code goes here
+  btnLeft.className = 'btn stop';
+  btnLeft.innerText = 'STOP';
 }
 
 function setSplitBtn() {
   // ... your code goes here
+  btnRight.className = 'btn split';
+  btnRight.innerText = 'SPLIT';
 }
 
 function setStartBtn() {
   // ... your code goes here
+  btnLeft.className = 'btn start';
+  btnLeft.innerText = 'START';
 }
 
 function setResetBtn() {
   // ... your code goes here
+  btnRight.className = 'btn reset';
+  btnRight.innerText = 'RESET';
 }
 
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
   // ... your code goes here
+  if (btnLeft.classList.contains('start')) {
+    setStopBtn();
+    setSplitBtn();
+    chronometer.startClick(printTime);
+  }
+  else {
+    setStartBtn();
+    setResetBtn();
+    chronometer.stopClick();
+  }
 });
 
 // Reset/Split Button
 btnRight.addEventListener('click', () => {
   // ... your code goes here
+  if (btnRight.classList.contains('reset')) {
+    setStartBtn();
```

Comment:
The right button doesn't change the button classes, so calling this functions here is not needed. Same in L107, L126 and L127.